.gitignore

```diff
@@ -1,137 +0,0 @@
-# Byte-compiled / optimized / DLL files
```

Comment:
Sorry for the confusion, could you revert this file? What I meant before was to remove only the line that you have added!